homeassistant/components/bond/config_flow.py

```diff
@@ -113,10 +112,11 @@ async def async_step_user(
         )
 
     async def _try_create_entry(self, data: Dict[str, Any]) -> Dict[str, Any]:
-        bond_id = await _validate_input(data)
+        bond_id, name = await _validate_input(data)
         await self.async_set_unique_id(bond_id)
         self._abort_if_unique_id_configured()
-        return self.async_create_entry(title=bond_id, data=data)
+        hub_name = f"{name} ({bond_id})" if name else bond_id
```

Comment:
Same
```suggestion
        hub_name = name or bond_id
```

Reply:
Definitely a case of I am not the user :)